src/test/java/io/xpring/UtilsTest.java

```diff
@@ -9,19 +9,23 @@
  * Unit tests for {@link Utils}.
  */
 public class UtilsTest {
-
     @Test
-    public void testIsValidAddressValidAddress() {
+    public void testIsValidAddressValidClassicAddress() {
         assertTrue(Utils.isValidAddress("rU6K7V3Po4snVhBBaU29sesqs2qTQJWDw1"));
     }
 
+    @Test
+    public void testIsValidAddressValidXAddress() {
+        assertTrue(Utils.isValidAddress("XVLhHMPHU98es4dbozjVtdWzVrDjtV18pX8yuPT7y4xaEHi"));
+    }
+
     @Test
     public void testIsValidAddressInvalidAlphabet() {
         assertFalse(Utils.isValidAddress("1EAG1MwmzkG6gRZcYqcRMfC17eMt8TDTit"));
     }
 
     @Test
-    public void testIsvValidAddressInvlalidChecksum() {
+    public void testIvValidAddressInvalidClassicAddressChecksum() {
```

Comment:
I think you were intending to fix a typo but you still have one here: `testIvValid...` instead of `testIsValid...`